Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -2054,6 +2065,14 @@ theorem le_mk_iff_exists_subset {c : Cardinal} {α : Type u} {s : Set α} :
   rw [le_mk_iff_exists_set, ← Subtype.exists_set_subtype]
   apply exists_congr; intro t; rw [mk_image_eq]; apply Subtype.val_injective
 
+@[simp]
+theorem mk_range_inl : #(range (@Sum.inl α β)) = #α :=
```

Comment:
Do you assume that `α` and `β` are in the same universe here? Should we use `lift` in the RHS instead?

Reply:
Nice catch! I've generalized both statements.

I didn't find `mk_range_eq` any easier to use than to just use the equiv.